server/webhook.go

```diff
@@ -210,6 +210,14 @@ func (s *Server) registerWebhookRoutes(g *echo.Group) {
 					},
 				)
 			}
+			for _, modified := range commit.Modified {
+				files = append(files,
+					fileItem{
+						name:     modified,
+						itemType: fileItemTypeModified,
+					},
+				)
+			}
 
 			if repo.Project.SchemaMigrationType == api.ProjectSchemaMigrationTypeSDL {
```

Comment:
You can just remove the line 222 (new) instead.

Reply:
Oops